UK_Reports/static/src/css/UK_Reports-body.css

```diff
@@ -11,10 +11,10 @@ table tr.row-border-bottom {
 	border-bottom: 1px solid;
 }
 table tr th.wide {
-	width: 500px;
```

Comment:
Maybe the changes need to backporting to 10.0, but there was not an reported issue with the reports format when they were printed. I made these changes because in version 13.0 there report was not looking good, and showed to Narender after made these changes.

Reply:
OK, just wanted to check they weren't things which were required on all branches, it seems Odoos reporting engine has most likely changed slightly between 10 and 13, so I think it's fine to just have these changes in 13